default_group_hosts: ["mpc01","mpc02","mbot01n","mbot02n","mbot03n","mbot04n","mbot05n","mbot06n","mbot07n","mbot08n","mbot09n","mbot10n","mbot11n","mbot12n","mbot13n","mbot14n"]